controlplane/kubeadm/controllers/fakes_test.go

```diff
@@ -57,20 +55,6 @@ func (f *fakeManagementCluster) GetMachinesForCluster(c context.Context, n clien
 	return f.Machines, nil
 }
 
-func (f *fakeManagementCluster) TargetClusterControlPlaneIsHealthy(_ context.Context, _ client.ObjectKey) error {
-	if !f.ControlPlaneHealthy {
-		return errors.New("control plane is not healthy")
-	}
-	return nil
-}
-
-func (f *fakeManagementCluster) TargetClusterEtcdIsHealthy(_ context.Context, _ client.ObjectKey) error {
-	if !f.EtcdHealthy {
```

Comment:
Similar comment of L61 /  `ControlPlaneHealthy`

Reply:
Same as above. Will do the cleanup.

Reply:
I found one case that this was used to added them to fake workload cluster.